### PR TITLE
recipeasy fixes

### PR DESCRIPTION
**Pull request by tsop14**
## What does this change?

- Adds comments to ingredients
- Changes time to mins

## What is the value of this and can you measure success?
Corrects data markup. 

## Does this affect other platforms - Amp, Apps, etc?
No.

## Screenshots
screen shot 2017-03-27 at 15 06 15

## Tested in CODE?
No.

Thanks @duarte for spotting 👀

### CODE REVIEW

**Comment by shtukas** [comment]
:+1:

File: common/app/model/content/Atoms.scala [hunk 1]
```diff
@@ -331,7 +331,8 @@ object RecipeAtom {
       .map(formatQuantity)
       .orElse(ingredient.quantityRange.map(range => s"${formatQuantity(range.from)}-${formatQuantity(range.to)}" ))
       .getOrElse("")
-    s"""${q} ${formatUnit(ingredient.unit.getOrElse(""))} ${ingredient.item}"""
+    val comment = ingredient.comment.fold("")(c => s", $c")
+    s"""${q} ${formatUnit(ingredient.unit.getOrElse(""))} ${ingredient.item}${comment}"""
```
Comment (LATaylor-guardian) [inline common/app/model/content/Atoms.scala]: Should there be any punctuation or spaces between ${ingredient.item} and ${comment}

**Comment by LATaylor-guardian** [comment]
Looks good to me, merge it 👍

**Comment by tomrf1** [comment]
👍

File: common/app/model/content/Atoms.scala (hunk 1 above)
Comment (tsop14) [inline common/app/model/content/Atoms.scala]: The punctuation (comma) is added if there is a value i.e `", $c"`

**Pull request merged**